### PR TITLE
LPS-187934 DM Site Settings - added section for Copy To Max Limit

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
After the changes: 

Screenshot 2023-06-19 at 09 36 06

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
ci:test:sf

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -15711,6 +15712,8 @@ sites-tab-description=Sites can only be assigned to a single property at a time.
 sites-that-i-administer=Sites That I Administer
 sites-to-sync-x=Sites to Sync - {0}
 size=Size
+size-limit-copy-files-title=Size limit for copying files
+size-limit-copy-files-help=Set the maximum size for copying files in bytes. The total number of copied files must not exceed the value specified in this property to complete the copying.
```
Comment (adolfopa) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:15716]: I'd change this to: "Set the maximum file size in bytes for copying files. The total size of all copied files must not exceed this value."

Comment (ambrinchaudhary) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:15716]: Changed! Thanks

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
ci:reevaluate:1317401_8449

**Comment by adolfopa** [comment]
See https://github.com/liferay-lima/liferay-portal/pull/4403

**Pull request closed without merging**